interface.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 	// If not set, it uses all versions available.
 	// Warning: This API should not be considered stable and will change soon.
 	Versions []protocol.VersionNumber
+	// Ask the server to truncate the connection ID sent in the Public Header.
```

Comment:
Please explain why a user would enable this (space savings), and the requirements for this option to be safe.